Update README.md (#26207)
kubernetes/website: README.md

make container-serve
```

> Open up your browser to http://localhost:1313/docs/reference/kubernetes-api/ to view the API reference pages.

Reviewer: ```suggestion
In a web browser, go to http://localhost:1313/docs/reference/kubernetes-api/ to view the API reference.
```
